cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/OperationalLimitConversion.java

```diff
@@ -192,7 +192,8 @@ public boolean valid() {
 
     @Override
     public void convert() {
-        double value = p.asDouble("value");
+        double normalValue = p.asDouble("normalValue");
+        double value = p.asDouble("value", normalValue);
```

Comment:
I don't understand the conversion here. The limits could be in units or in percentages. In the second case, we need the normal value to convert the limit in unit in IIDM. Where have we that conversion?

Reply:
In CGMES 3 values for limits have been moved from EQ to SSH. EQ now contains `normalValue`, that should be used when no explicit `value` is defined in SSH. That is, `normalValue` in EQ is the default value for the limit if a value is not explicitly given in the SSH.

For Operational Limits, all the limit values and normal values are given in the corresponding engineering units, according to the specification. I think is for Measurements that there is the possibility of providing values in defined units or as percentages of normal values.